spec/examples/to_yaml_spec.rb

```diff
@@ -78,7 +78,7 @@ module IceCube
     end
 
     it "should be able to make a round-trip to YAML with .day_of_year" do
-      schedule1 = Schedule.new(Time.now)
+      schedule1 = Schedule.new(Time.zone.now)
```

Comment:
`Time.zone` is another active support [extension](https://github.com/rails/rails/blob/main/activesupport/lib/active_support/core_ext/time/zones.rb#L14). The stock `Time` class in Ruby has no `zone` method.